README.md

```diff
@@ -4,18 +4,19 @@ A simple statically-generated site, implemented in [Metalsmith](http://www.metal
 
 Currently in beta.
 
-
 ### Running locally
 
 - Clone the repo
 - `npm install && bower install`
 - Create a new `.env` file at the project root. (e.g. `touch .env && nano .env`)
 - Add the following keys to `.env` (You'll need to contact the project admin privately for API keys):
+
 ```
 ENV=development
-CONTENTFUL_ACCESS_TOKEN=<contentful access token>
+CONTENTFUL_PREVIEW_ACCESS_TOKEN=<contentful access token>
```

Comment:
I think this is only correct if you use `yarn serve` (which from the comments it looks like you did, and if that still works, I'm amazed, because that's been deprecated/unmaintained since like 2016). If you're building the site as normal (e.g. with `yarn build`), I'm pretty sure this still needs to be `CONTENTFUL_ACCESS_TOKEN`.